av1_codec_registration.src.html

```diff
@@ -74,6 +74,47 @@
 contain a frame with a `frame_type` of `KEY_FRAME` as defined in Section
 6.8.2 of [[AV1]].
 
+VideoEncoderConfig extensions {#videoencoderconfig-extensions}
```

Comment:
If I understand the structure of this document. VideoEncoderConfig needs to be added to the 'Abstract' section as well.

Reply:
Fixed.